pkg/ccip/bindings/offramp/offramp.go

```diff
@@ -47,6 +69,90 @@ type Storage struct {
 	LatestPriceSequenceNumber               uint64              `tlb:"## 64"`
 }
 
+// Deployables holds the deployable code cells for the offRamp contract
+type Deployables struct {
+	Deployer            *cell.Cell `tlb:"^"`
+	MerkleRootCode      *cell.Cell `tlb:"^"`
+	ReceiveExecutorCode *cell.Cell `tlb:"^"`
+}
+
+// ConfigInfo represents the configuration information for OCR3
+type ConfigInfo struct {
+	ConfigDigest                   []byte `tlb:"bits 256"`
+	F                              uint8  `tlb:"## 8"`
+	N                              uint8  `tlb:"## 8"`
+	IsSignatureVerificationEnabled bool   `tlb:"bool"`
+}
+
+// Methods
+
+// UpdateSourceChainConfig represents the updateSourceChainConfig method call on the offRamp contract
+type UpdateSourceChainConfig struct {
```

Comment:
nit: Let's try to provide some order to these binding types (e.g., all msg structs together, input first, output next, other data structures next, etc.)